src/lib/stores.ts

```diff
@@ -1,10 +1,53 @@
 import { writable } from 'svelte/store';
-import { ActionForms, defaultDotApi } from '$lib/storeTypes';
+import { ActionForms, defaultDotApi, type DotApi } from '$lib/storeTypes';
+import { storable } from './stores/storable';
+import { derived } from 'svelte/store';
+import { user } from './stores/userStore';
+import { createApi, getToken } from './polkadotApi';
+import { getBlockNumber, getEpoch } from './connections';
+import { pageContent } from './stores/pageContentStore';
 
-export const dotApi = writable(defaultDotApi);
+export const dotApi = writable<DotApi>(defaultDotApi);
 
 export const storeCurrentAction = writable(ActionForms.NoForm);
 
-export const storeChainInfo = writable({ connected: false, blockNumber: 0n, epochNumber: 0n, token: '' });
+export const isLoggedIn = storable<boolean>('isLoggedIn', false);
```

Comment:
Hmm, maybe I lack context, but this seems like something that you might want in session storage rather than local storage...

Reply:
What would the difference be? Curious why.

Reply:
My other argument is deleting it and just checking that a user is connected.

Reply:
Session storage is different for different tabs, and is cleared when the tab is closed. I would think that a good fit for a login session indicator.

Reply:
@claireolmstead let's track this in a separate issue; post the issue link here & then we can resolve this.

Reply:
https://github.com/LibertyDSNP/provider-dashboard/issues/154